.circleci/config.yml

```diff
@@ -15,6 +15,13 @@ jobs:
       - run:
           name: Install Node Modules
           command: npm install
+      - run:
+          name: Download Selenium
+          command: curl -O http://selenium-release.storage.googleapis.com/3.5/selenium-server-standalone-3.5.3.jar
+      - run:
+          name: Start Selenium
+          command: java -jar selenium-server-standalone-3.5.3.jar -log test-reports/selenium.log
+          background: true
```

Comment:
I think these should go town in the `test:` section (around line 38).

I also realized that because Jest isn't running in CI (#992), and the current functional tests are written with Jest, that this will make it hard to include Selenium. The issue in #992 is... somewhat mysterious to me, something about Jest and things running somewhat differently in CI (different Node version?)

Reply:
Okay fixing it right away. So I figured what was making my branches orphaned. I won't need to be closing more branhes